wordle-web/components/SettingsDialog.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-dialog v-model="modelValue" max-width="500">
+    <v-card>
+      <v-sheet color="primary">
+        <v-card-text> Settings </v-card-text>
+      </v-sheet>
+      <v-card-text>
+        Dark/Light:   
+        <v-btn 
+          icon="mdi-theme-light-dark" @click="toggleTheme" />
+      </v-card-text>
+      
+      <v-sheet class="ml-6">
+        <v-select
+          color="secondary"
+          v-model="chosenTheme"
+          label="Choose a Theme"
+          :items="['Standard', 'Mystic Meadow', 'Enchanted Eclipse']"
+        ></v-select>
```

Comment:
```suggestion
        <v-select
          color="secondary"
          v-model="chosenTheme"
          label="Choose a Theme"
          :items="['Standard', 'Mystic Meadow', 'Enchanted Eclipse']"
        />
```
No content inside v-select so you can use self-closing tag

Reply:
Fixed, thank you!